docs/deployment/digitalocean.md

```diff
@@ -0,0 +1,136 @@
+# How to deploy appwrite into DigitalOcean
+
+## Prequisites
+
+- OS 
+- Terminal 
+
+## Getting started
+
+First of all you should get a DigitalOcean account, if you already got an account let's continue to setup our appwrite server.
+
+## Step 1 - Create a Droplet
+
+What is a Droplet? DigitalOcean Droplet are Linux-based virtual machines (VMs) that run on top of virtualized hardware. Each Droplet you create is a new server you can use, either standalone or as part of a larger, cloud-based infrastructure.
+ This droplet then can be used to run our appwrite server
```

Comment:
Random space at the beginning of the line here

Reply:
fixed in 10396a29017fdce59f2247fc567994df4608c0c4